_sub/compute/helm-ebs-csi-driver/vars.tf

```diff
@@ -21,7 +21,14 @@ variable "cluster_name" {
   description = "The cluster name"
 }
 
-variable "kiam_server_role_arn" {
+variable "csi_ebs_serviceaccount_name" {

```

Comment:
Why are these variables, when they are hardcoded and not even used nor exposed in k8s-services. Could they be either locals or be actual referenced in a set block for the helm chart?